Sources/PackageDescription/PackageDescriptionSerialization.swift

```diff
@@ -429,6 +495,16 @@ extension SwiftVersion: Encodable {
 }
 
 extension Version: Encodable {
+    /// Encodes this value into the given encoder.
+    ///
+    /// If the value fails to encode anything, `encoder` will encode an empty
+    /// keyed container in its place.
+    ///
+    /// This function throws an error if any values are invalid for the given
+    /// encoder's format.
+    ///
+    /// - Parameters:
+    ///   - encoder: The encoder to write data to.
```

Comment:
I don't think this description is accurate.

Reply:
I'll talk with Anders more about this. This file is really an internal implementation detail, but Anders thought it was not possible to hide it from DocC. But you are correct, the documentation should be accurate.